src/Generators/Webserver/Database/DatabaseGenerator.php

```diff
@@ -103,7 +80,7 @@ public function created(Events\Websites\Created $event)
             new Events\Database\Creating($config, $event->website)
         );
 
-        if (!$this->driver($config)->created($event, $config, $this->connection)) {
+        if (!app(DatabaseDriverFactory::class)->create($config['driver'])->created($event, $config, $this->connection)) {
```

Comment:
There's no reason to use `app` here to resolve a class. I'd prefer you add the dependency in the construct. Because the factory is needed on all methods anyway.

Reply:
Done.